packages/launcher/lib/browsers.ts

```diff
@@ -166,6 +166,7 @@ export function launch (
   debuggingPort: number,
   args: string[] = [],
   defaultBrowserEnv = {},
+  launchOptionsEnv = {},
```

Comment:
This is too many parameters for this function, can you either remove the need for this parameter or move the parameters for `.launch` into an `options` object?

Reply:
:+1: I'll collapse the two env members into one.